scss/components/_blog.scss

```diff
@@ -28,8 +28,8 @@
   color: $black;
 }
 .blog-folders {
-  margin-left: 115px;
-  width: 100%;
+  width: 84%;
```

Comment:
Do not use `width: 84%;`. Use only for 100%, 50%, etc

Reply:
Here and in all other cases.

Reply:
Got it